src/controllers/deleteGuildRoleHandler.ts

```diff
@@ -0,0 +1,42 @@
+import { IRequest } from "itty-router";
+import JSONResponse from "../utils/JsonResponse";
+import { env } from "../typeDefinitions/default.types";
+import * as response from "../constants/responses";
+import { verifyNodejsBackendAuthToken } from "../utils/verifyAuthToken";
+import { deleteGuildRole } from "../utils/deleteGuildRole";
+
+export async function deleteGuildRoleHandler(request: IRequest, env: env) {
+  const authHeader = request.headers.get("Authorization");
+  const reason = request.headers.get("X-Audit-Log-Reason");
+  const roleId = decodeURI(request.params?.roleId ?? "");
+  const { dev } = request.query;
+  const devFlag = dev === "true" ? true : false;
```

Comment:
Using the ternary operator is redundant here
```suggestion
  const devFlag = dev === "true";
```